packages/components/src/context-actions/GlobalContextAction.tsx

```diff
@@ -1,12 +1,22 @@
 import { Component } from 'react';
-import PropTypes from 'prop-types';
 import Log from '@deephaven/log';
-import ContextActionUtils from './ContextActionUtils';
+import ContextActionUtils, {
+  ContextAction,
+  ContextActionEvent,
+  KeyState,
+} from './ContextActionUtils';
 
 const log = Log.module('GlobalContextAction');
 
-class GlobalContextAction extends Component {
-  constructor(props) {
+interface GlobalContextActionProps {
+  action: ContextAction;
+}
+
+class GlobalContextAction extends Component<
+  GlobalContextActionProps,
+  KeyState
```

Comment:
I don't reaaaaally like using `KeyState` directly here, should define a `type GlobalContextActionState = KeyState` above and use that.